tasks/21_check-letters/check-letters.js

```diff
@@ -0,0 +1,13 @@
+export function checkLetters([firstArray, secondArray]) {
```

Comment:
`secondArray` is a misleading name since the second argument is actually a string.